scripts/check-file-size.sh

```
@@ -0,0 +1,87 @@
+      app/\(app\)/app/questions/\[slug\]/question-page-client.tsx)
+      return 0
+      ;;
+    *)
+      return 1
+      ;;
+  esac
+}
+
+is_test_file() {
+  case "$1" in
+    *.test.ts | *.test.tsx | *.spec.ts | *.spec.tsx | *.browser.spec.tsx)
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 22.

(SC2221)